internal/api/openapi/rhtas-console.yaml

```diff
@@ -448,15 +448,14 @@ components:
           description: >
             URL of a TUF repository containing the trusted root JSON file.
           example: https://tuf-repo-cdn.sigstore.dev
-          nullable: true
         predicateType:
           type: string
           description: >
             The type of the predicate for the attestation.
           example: https://example.com/attestations/build
       required:
-        - expectedIssuer
-        - expectedSAN
+        - ociImage
+        - tufRootURL
```

Comment:
For the UI don't we always assume the same TUF root URL? Is there a reason we need to introduce this required property?